requirements/app/app.txt

```diff
@@ -1,30 +1,30 @@
 lightning-cloud >=0.5.37
 packaging
-typing-extensions >=4.0.0, <=4.7.1
+typing-extensions >=4.0.0, <4.8.0
 deepdiff >=5.7.0, <6.3.2
 starsessions >=1.2.1, <2.0 # strict
-fsspec >=2022.5.0, <=2023.6.0
+fsspec >=2022.5.0, <2023.7.0
 croniter >=1.3.0, <1.5.0  # strict; TODO: for now until we find something more robust.
 traitlets >=5.3.0, <5.10.0
-arrow >=1.2.0, <1.2.4
+arrow >=1.2.0, <1.3.0
 lightning-utilities >=0.8.0, <0.10.0
-beautifulsoup4 >=4.8.0, <4.12.3
-inquirer >=2.10.0, <=3.1.3
+beautifulsoup4 >=4.8.0, <4.13.0
+inquirer >=2.10.0, <=3.2.0
```

Comment:
Why have you left some dependencies with "<="? Don't you want to update all?

Reply:
```suggestion
inquirer >=2.10.0, <3.2.0
```